src/cc_helpers.cc

```diff
@@ -338,6 +353,18 @@ bool certifier::framework::cc_trust_manager::initialize_oe_enclave(
 #endif
 }
 
+bool certifier::framework::cc_trust_manager::accelerator_verified(const string& acc_type) {
+  for (int i = 0; i < num_accelerators_; i++) {
+    if (acc_type == accelerators_[i].accelerator_type_)
```

Comment:
what if we have several accelerators of the same type? for example several NVIDIA H100